Sources/Library/Controllers/UsersController.swift

```diff
@@ -1,24 +1,25 @@
 import Vapor
 import HTTP
 
-final class UsersController: ResourceRepresentable {
-    
-    func index(request: Request) throws -> ResponseRepresentable {
+public final class UsersController: ResourceRepresentable {
+    public init() {}
+
+    public func index(request: Request) throws -> ResponseRepresentable {
         let json = try JSON(node: User.all().makeNode())
         return json
     }
-    
-    func store(request: Request) throws -> ResponseRepresentable {
+
+    public func store(request: Request) throws -> ResponseRepresentable {
         var user = try request.user()
         try user.save()
         return user
     }
-    
-    func show(request: Request, user: User) throws -> ResponseRepresentable {
+
+    public func show(request: Request, user: User) throws -> ResponseRepresentable {
```

Comment:
Yes, if you're building a library, unless everything is marked public, these symbols are not accessible outside of the library.